packages/zoe/src/zoeService/types.js

```diff
@@ -39,8 +39,12 @@
  * Return an object with the instance, installation, description, invitation
  * handle, and any custom properties specific to the contract.
  * @property {GetFeeIssuer} getFeeIssuer
- * @property {MakeFeePurse} makeFeePurse
- * @property {BindDefaultFeePurse} bindDefaultFeePurse
+ * @property {() => Promise<Purse>} makeFeePurse
+ * Deprecated. Does nothing useful but provided during transition so less old
+ * code breaks.
+ * @property {(defaultFeePurse: ERef<Purse>) => ZoeService} bindDefaultFeePurse
```

Comment:
Isn't it gone now? Can't this be dropped?  What still refers to it?

Reply:
Nothing in agoric-sdk, but other dapps still do, so I'm leaving in this placeholder as a transitional measure. Resolving without change.

(I actually started out removing it, but then CI slapped be with dapp failures. As it should!)